### ISSUE TITLE
Consider only including `yes` values in roads layer conditional properties

### ISSUE THREAD

**Issue by rmarianski** [mapzen/vector-datasource#249]
In the roads layer, `is_link`, `is_tunnel`, and `is_bridge` properties are included on all features. To trim down on file sizes, how about we only include the property when it's true?

**Comment by nvkelso** [comment]
+1

**Comment by rmarianski** [comment]
@bcamper, @meetar: would this be disruptive to some styles?

**Comment by meetar** [comment]
Most of the time I've been filtering on "yes" values explicitly, and very rarely on things like "tunnel: false" -- so worst case this would be mildly disruptive with easy fixes, best case is not at all!

**Comment by rmarianski** [comment]
@nvkelso once this has been deployed to dev, can we check some demos/styles that use the roads layer to ensure that they aren't broken? In particular, ones that make use of the properties in the issue description?

**Comment by meetar** [comment]
I'll test it out on https://github.com/tangrams/highways-demo, just give me the signal :traffic_light:

**Comment by nvkelso** [comment]
@Rob I'm still seeing `"oneway":"no"` (which seems like we should drop) in tiles like: 

* http://vector.dev.mapzen.com/osm/all/14/4826/6161.topojson

Separately, I see `"motor_vehicle":"no"` in that same tile. Should we be preserving or dropping that? @zerebubuth what do you think?

**Comment by nvkelso** [comment]
(We can spin the `oneway` and `motor_vehicle` questions into v0.7 release if this is a hassle.)

**Comment by zerebubuth** [comment]
> I'm still seeing "oneway":"no" (which seems like we should drop)

For consistency, yes, I guess we should be dropping that too. I think there are some road types which imply `oneway=yes` by default, but we should do some checking. I've added #453 to track this in v0.7.0.

> Separately, I see "motor_vehicle":"no" in that same tile. Should we be preserving or dropping that?

We should be preserving this: the default for most roads is `motor_vehicle: yes`, so if we normalised to dropping `motor_vehicle: no` then we'd have to add in a lot of `motor_vehicle: yes` where the property is currently omitted to ensure that the information remained. This is possible, but the [implicit defaults](http://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access-Restrictions) vary slightly between countries and normalising them would be a large task.
